docker/docker-compose.yml

```diff
@@ -46,10 +46,10 @@ services:
     container_name: jetpack_sftp
     image: atmoz/sftp
     volumes:
-      - ..:/home/wordpress/var/www/html/wp-content/plugins/jetpack
+      ## WordPress and its plugins and themes should be defined in docker/compose-volumes.yml{.sample}, not here!
+      ## That avoids haing to define them both here and in the "wordpress" service.
```

Comment:
```suggestion
      ## That avoids having to define them both here and in the "wordpress" service.
```

Reply:
Thanks! I'll squash this into the update I'm about to push for the other comments.